nitorch/nn/training/_training.py

```diff
@@ -563,15 +574,20 @@ def train(self):
                 self._eval(self.epoch)
                 self._save(self.epoch)
                 for self.epoch in range(self.epoch+1, self.nb_epoch+1):
-                    self._train(self.epoch)
-                    self._eval(self.epoch)
+                    _ = self._train(self.epoch)
+                    val_loss = self._eval(self.epoch)
```

Comment:
There is not always a validation set! I don’t know if you should give it the training loss, or the eval loss if it exists else the training loss?

Reply:
Thanks for catching that, fixed.